fossunited/ticketing/doctype/foss_event_ticket_transfer/test_foss_event_ticket_transfer.py

```diff
@@ -1,8 +1,119 @@
 # Copyright (c) 2024, Frappe x FOSSUnited and Contributors
 # See license.txt
 
+from datetime import datetime, timedelta
+
+import frappe
 from frappe.tests.utils import FrappeTestCase
 
 
 class TestFOSSEventTicketTransfer(FrappeTestCase):
-    pass
+    def test_ticket_transfer(self):
+        # Given for an event, a ticket is created. For that ticket, a transfer is generated.
+        event = frappe.get_doc(
+            {
+                "doctype": "FOSS Chapter Event",
+                "event_name": "Test Event",
+                "event_permalink": "test_perma",
+                "status": "Approved",
+                "event_type": "Conference",
+                "event_start_date": datetime.today(),
+                "event_end_date": datetime.today() + timedelta(1),
+                "event_description": "testing",
+            }
+        )
+        event.insert()
+
+        ticket = frappe.get_doc(
+            {
+                "doctype": "FOSS Event Ticket",
+                "event": event.name,
+                "full_name": "Harsh Tandiya",
+                "email": "harsh@test.xyz",
+            }
+        )
+        ticket.insert()
+
+        transfer = frappe.get_doc(
+            {
+                "doctype": "FOSS Event Ticket Transfer",
+                "ticket": ticket.name,
+                "receiver_name": "Rahul",
+                "receiver_email": "rahul@test.xyz",
+            }
+        )
+        transfer.insert()
+
+        # When Ticket Transfer is created, status should be pending
+        self.assertEqual(transfer.status, "Pending Approval")
+
+        # Change status of transfer to completed
+        transfer.status = "Completed"
+        transfer.save()
+        self.assertEqual(transfer.status, "Completed")
```

Comment:
You don't have to test the framework 😅

Reply:
this might be a left over assert - at first, we were using a method on the doctype to set the status and perform the validation internally. we ran into some issues there so this assert can go now.